### ISSUE TITLE
Feature Request: Add Screenshots for FloatingActionButtonLocation

### ISSUE THREAD

**Issue by pennzht** [flutter/flutter#55321]
## Use case

Once merged, PR #51465 will **enable many new FAB Locations** and **introduce ways to create user-defined FAB Locations**.

Screenshots in documentation will be very helpful for developers.

## Proposal

We propose that:
* We add a small screenshot to each FAB Location, illustrating its effect and typical usage.
* We add diagrams to illustrate what each mixin does.

**Comment by Piinks** [comment]
Indeed! https://api.flutter.dev/flutter/material/FloatingActionButtonLocation-class.html
Thanks! I will close this then.
